learn/templates/404.html

```diff
@@ -1,4 +1,619 @@
+            color: rgba(255, 255, 255, 1);
+            font-weight: 400;
+        }
+
+
+
+        /* === SEARCH FORM === */
+
+        #search {
+            margin-top: 30px;
+        }
+
+        #search input[type='text'] {
+            width: 88%;
+            height: 41px;
+            padding-left: 15px;
+            padding-rigt: 15px;
+            box-sizing: border-box;
+            -moz-box-sizing: border-box;
+            background-color: rgba(0, 0, 0, 0.2);
+            border: none;
+            outline: none;
+            -webkit-appearance: none;
+            font-size: 16px;
+            font-weight: 300;
+            color: white;
+            font-family: 'Source Sans Pro', sans-serif;
+            transition: all 0.5s ease;
+            -moz-transition: all 0.5s ease;
+            -o-transition: all 0.5s ease;
+            -webkit-transition: all 0.5s ease;
+            border-radius: 0px;
+        }
+
+        #search .input-search {
+            width: 10%;
+            float: right;
+            height: 41px;
+            background-color: rgba(0, 0, 0, 0.2);
+            outline: none;
+            border: none;
+            -webkit-appearance: none;
+            font-family: 'Source Sans Pro', sans-serif;
+            color: white;
+            font-weight: 300;
+            font-size: 16px;
+            cursor: pointer;
+            transition: all 0.5s ease;
+            -moz-transition: all 0.5s ease;
+            -o-transition: all 0.5s ease;
+            -webkit-transition: all 0.5s ease;
+            text-align: center;
+        }
+
+        #search .input-search:hover {
+            background-color: rgba(26, 188, 156, 0.7);
+        }
+
+
+
+        /* === RESPONSIVE CSS === */
+
+        @media all and (max-width: 899px) {
+            #wrap {
+                width: 90%;
+            }
+        }
+
+        @media all and (max-width: 799px) {
+            #wrap {
+                width: 90%;
+                height: auto;
+                margin-top: 40px;
+                top: 0%;
+            }
+
+            #wordsearch {
+                width: 90%;
+                float: none;
+                margin: 0 auto;
+            }
+
+            #wordsearch ul li {
+                font-size: 4vw;
+            }
+
+            #main-content {
+                float: none;
+                width: 90%;
+                max-width: 90%;
+                margin: 0 auto;
+                margin-top: 30px;
+                text-align: justify;
+            }
+
+            #main-content h1 {
+                text-align: left;
+            }
+
+            #search input[type='text'] {
+                width: 84%;
+            }
+
+            #search .input-search {
+                width: 15%;
+            }
+        }
+
+        @media all and (max-width: 499px) {
+            #main-content h1 {
+                font-size: 28px;
+            }
+        }
+    </style>
+    <title>Start Learning</title>
```

Comment:
Page title should be something more relevant like `Easy Learning - 404 Page Not Found`

Reply:
Okay